src/core/components/SearchChatHistory.module.css

```diff
@@ -15,10 +14,6 @@
   transition: color 0.2s ease;
 }
 
-.iconButton:hover {
-  color: var(--colors-textMuted);
-}
-
```

Comment:
Don't need to explicitly change color on hover now that we've got base ButtonIcon styles 🦾